spec/features/user_spec.rb

```diff
@@ -1,12 +1,56 @@
 require 'spec_helper'
 
 feature User do
+  let(:user) { create(:user, name: 'My Name', biography: 'This is my biography!') }
 
-  shared_examples 'admin ability' do
+  describe 'users#show' do
+    let!(:conference) { create(:conference, title: 'My conference') }
+    let!(:unconfirmed_event) { create(:event, state: 'unconfirmed', title: 'This is my test event title!', program: conference.program) }
 
-  end
+    before :each do
+      unconfirmed_event.event_users = [create(:event_user, user: user, event_role: 'submitter')]
```

Comment:
let!(:unconfirmed_event) { create(:event, state: 'unconfirmed', title: 'This is my test event title!', program: conference.program, users: [create(:event_user, user: user, event_role: 'submitter')]) }
does not work?